draft-pda-core.md

```diff
@@ -147,17 +146,21 @@ because modular addition is commutative. I.e.,
       = F(x[1], x[2], x[3])
 ```
 
-This is essentially how all Prio computations are performed: measurements
-are encoded into in a manner that allows the function F to be expressed as a sum of
-the aggregators' shares of the aggregate; clients split their encoded values into
-shares, sending one share to each server; the servers add up their shares;
-and the servers combine their aggregate shares to get the final output of F. Not
-all aggregate functions can be expressed this way, however. Prio supports a
-limited set of aggregation functions, some of which we highlight below:
-
-- Simple statistics, like sum, mean, min, max, variance, and standard
-  deviation; [[OPEN ISSUE: It's possible to estimate quantiles such as the
-  median. How practical is this?]]
+**Prio.**
+This approach can be used to privately compute any function `F` that can be
+expressed as a function of the sum of the users' inputs. In Prio {{BC17}}, each
+user encodes its measurement as a sequence of elements over some prime field. It
+then splits its input into shares and sends each share to the aggregators. The
+aggregators sum up their input shares. Once all the shares have been aggregated,
+they combine their shares of the aggregate to get the final output.
```

Comment:
I think we could do without the "prime field" bit here. It's not necessary to explain what Prio is for.

Reply:
What would you put instead?

Reply:
I would remove the entire sentence.

Reply:
That works.